Jobim2020 (#420)
nf-core/nf-co.re: markdown/events/2020/jobim-2020.md

> The talk will be available for anyone to watch after the conference.
> It will be posted on this page when ready.

Reviewer: ```suggestion
The talk will be available for anyone to watch after the conference.
You can see the slides below.
```

> <div class="row">
>     <div class="col-md-6">
>         <iframe src="https://maxulysse.github.io/jobim2020" width="595" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe>
>         <div style="margin-bottom:5px"> <strong> <a href="https://maxulysse.github.io/jobim2020" title="nf-core, a community effort for collaborative, peer-reviewed analysis pipelines" target="_blank">nf-core, a community effort for collaborative, peer-reviewed analysis pipelines</a> </strong> from <strong><a href="https://maxulysse.github.io" target="_blank">Maxime Garcia</a></strong> </div>
>     </div>
> </div>

Reviewer: Nice! Two minor suggestions:

* I don't think that we need the `col-md-6` to limit it to half of the width of the screen? There's nothing on the right so it just gives a bunch of whitespace
* The aspect ratio felt a little odd to me. Upping the width to `700` feels better

```suggestion
<iframe src="https://maxulysse.github.io/jobim2020" width="700" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe>
<div style="margin-bottom:5px"> <strong> <a href="https://maxulysse.github.io/jobim2020" title="nf-core, a community effort for collaborative, peer-reviewed analysis pipelines" target="_blank">nf-core, a community effort for collaborative, peer-reviewed analysis pipelines</a> </strong> by <strong><a href="https://maxulysse.github.io" target="_blank">Maxime Garcia</a></strong> </div>
```

![image](https://user-images.githubusercontent.com/465550/86492157-7d548f00-bd6d-11ea-86f7-e6841b9c8997.png)
